app/src/main/res/layout/activity_main.xml

```diff
@@ -8,13 +8,14 @@
     android:background="@drawable/image"
     android:orientation="vertical"
     android:padding="16dp">
+
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_gravity="center"
-        android:textColor="@android:color/white"
-        android:textSize="40sp"
-        android:text="Weather"/>
+        android:text="@string/app_title"
+        android:textColor="@color/title_color"
+        android:textSize="40sp" />
```

Comment:
What is the rationale for this change?  Describe and cite best practices here, inline, in the Files Changed section of the pull request.